### ISSUE TITLE
getting started

### ISSUE THREAD

**Issue by bbartling** [NubeDev/bacnet#10]
Hi,

So I am watching some go tutorials on the fly at the same time...

Do I clone this repo and then change directory into the cloned repo to get started? I at least have go installed on Ubuntu and can run a simple `hello world` go program.

**Comment by sirdethman** [comment]
Yeah I cant figure it out either. It says to use CD cli but there is not cli directory in the repo. Now I am new to GO so I may be doing something wrong.
